dashboard/lib/app.js

```diff
@@ -40,23 +40,38 @@ var express = require('express'),
 var app = express();
 
 
-logger.info('Running app in web context: '+config.web_context);
-app.base=config.web_context;
+logger.info('Running app in web context: ' + config.web_context);
```

Comment:
Use string interpolation instead of concatenation:
`logger.info('Running app in web context: %s', config.web_context)`